File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.cs
using System;

public class Solution 
{
    public string solution(string my_string, int num1, int num2) 
    {
        char[] str_list = my_string.ToCharArray();
        char temp = str_list[num1];
        str_list[num1] = str_list[num2];
        str_list[num2] = temp;
        string str = new string(str_list);
        return str;
    }
}

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int num1, int num2) 
{
    char tmp1 = my_string[num1];
    char tmp2 = my_string[num2];
    
    my_string[num1] = tmp2;
    my_string[num2] = tmp1;
    
    return my_string;
}